perl.spec

```diff
@@ -3,7 +3,7 @@
 %global perl_arch_stem -thread-multi
 %global perl_archname %{_arch}-%{_os}%{perl_arch_stem}
 
-%global multilib_64_archs x86_64 s390x ppc64 sparc64
+%global multilib_64_archs aarch64 %{power64} s390x sparc64 x86_64
```

Comment:
As we don't have multilib, shouldn't we just remove x86_64 instead? I also don't like replacing ppc64 with %{power64} when the commit message does not mention it at all.

Reply:
I think this change is ok. Maybe commit message could be adjusted to "[perl] Fix search libpath for 64bit arches."